views/view_package.dt

```diff
@@ -87,20 +93,40 @@ block body
 							|= depname
 						|= "- " ~ sp["description"].opt!string
 
-		hr
-		- if (readmeContents.length)
-			.repositoryReadme.markdown-body
-				- import vibe.textfilter.markdown : MarkdownFlags, MarkdownSettings, filterMarkdown;
-				- scope msettings = new MarkdownSettings;
-				- msettings.flags = MarkdownFlags.backtickCodeBlocks|MarkdownFlags.noInlineHtml|MarkdownFlags.tables;
-				- msettings.headingBaseLevel = 2;
-				- msettings.urlFilter = &urlFilter;
-				!= filterMarkdown(readmeContents, msettings)
+		.subtabsHeader
+			- if (readmeContents.length)
+				a.tab(class=activeTab == "info" ? "active" : "", href="#{req.rootDir}packages/#{normalizedPackageName}?tab=info") Info
+			- foreach (i, sample; sampleURLs)
+				a.tab(class=activeTab == "sample_"~i.to!string ? "active" : "", href="#{req.rootDir}packages/#{normalizedPackageName}?tab=sample_#{i}")= sample
+			a.tab.external(href="http://#{normalizedPackageName}.dpldocs.info/") Documentation
+		- bool renderedTabContent;
+		- if (activeTab == "info")
+			- if (readmeContents.length)
+				.repositoryReadme.markdown-body
+					- import vibe.textfilter.markdown : MarkdownFlags, MarkdownSettings, filterMarkdown;
+					- scope msettings = new MarkdownSettings;
+					- msettings.flags = MarkdownFlags.backtickCodeBlocks|MarkdownFlags.noInlineHtml|MarkdownFlags.tables;
+					- msettings.headingBaseLevel = 2;
+					- msettings.urlFilter = &urlFilter;
+					!= filterMarkdown(readmeContents, msettings)
+			- else
+				.repositoryReadme
+			- renderedTabContent = true;
```

Comment:
Another tab we could add would be stats e.g. like this:
https://github.com/dlang/dub-registry/issues/160

Reply:
Another one could be versions: http://code.dlang.org/packages/vibe-d/versions